articles/storage/blobs/secure-file-transfer-protocol-known-issues.md

```diff
@@ -81,6 +81,8 @@ To learn more, see [SFTP permission model](secure-file-transfer-protocol-support
 
 - For performance issues and considerations, see [SSH File Transfer Protocol (SFTP) performance considerations in Azure Blob storage](secure-file-transfer-protocol-performance.md).
   
+-  Blobs uploaded using SFTP doesn't calculate MD-5 even irrespective of any blob size.
```

Comment:
Thanks @Vignesh-AzurePaaS. I believe it's "MD5" instead of "MD-5". Also, the way that this phrase reads is that blobs somehow perform the calculation. It's more that the MD5 has is not calculated during upload and then placed in the Content-MD5 property correct? If so, I would modify slightly to "The Content-MD5 property of blobs that are uploaded by using SFTP don't contain an MD5 hash." Is this proposed rephrasing correct technically?